src/AssociationRegistry.Admin.Api/Verenigingen/Detail/ResponseModels/Vertegenwoordiger.cs

```diff
@@ -35,7 +35,7 @@ public class Vertegenwoordiger
     public bool PrimairContactpersoon { get; init; }
 
     /// <summary>Het emailadres van de vertegenwoordiger</summary>
-    [DataMember(Name = "Email")]
+    [DataMember(Name = "E-mail")]
```

Comment:
"hyphens" worden liever niet gebruikt in json omdat ze verward worden met "minus".